tests/phpunit/tests/block-bindings/render.php

```diff
@@ -234,4 +234,31 @@ public function test_source_value_with_unsafe_html_is_sanitized() {
 			'The block content should be updated with the value returned by the source.'
 		);
 	}
+
+	/**
+	 * Tests if the block content is sanitized when unsafe HTML is passed.
```

Comment:
Small nitpicks, the docblock doesn't match the source code.

The test description mention sanitizing unsafe HTML and the `@covers` doesn't match the method being tested (I believe it should be `\WP_Block::process_block_bindings`).

Reply:
Fix for this now pushed.